esp/src/src-react/components/controls/OverflowBreadcrumb.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { Overflow, Breadcrumb, OverflowItem, BreadcrumbItem, BreadcrumbButton, BreadcrumbButtonProps, BreadcrumbDivider, OverflowDivider } from "@fluentui/react-components";
+import { bundleIcon, Folder20Filled, Folder20Regular, FolderOpen20Filled, FolderOpen20Regular, } from "@fluentui/react-icons";
+import { OverflowMenu } from "./OverflowMenu";
+
+const LineageIcon = bundleIcon(Folder20Filled, Folder20Regular);
+const SelectedLineageIcon = bundleIcon(FolderOpen20Filled, FolderOpen20Regular);
+
+export interface BreadcrumbInfo {
+    id: string;
+    label: string;
+    props?: BreadcrumbButtonProps
+}
+
+interface OverflowGroupDividerProps {
+    groupId: string;
+}
+
+const OverflowGroupDivider: React.FunctionComponent<OverflowGroupDividerProps> = ({
+    groupId,
+}) => {
+    return <OverflowDivider groupId={groupId}>
+        <BreadcrumbDivider data-group={groupId} />
+    </OverflowDivider>;
+};
+
+function icon(breadcrumb: BreadcrumbInfo, selected: string) {
+    return breadcrumb.id === selected ? <SelectedLineageIcon /> : <LineageIcon />
+}
+
+export interface OverflowBreadcrumbProps {
+    breadcrumbs: BreadcrumbInfo[];
+    selected: string;
+    onSelect: (tab: BreadcrumbInfo) => void;
+}
+
+export const OverflowBreadcrumb: React.FunctionComponent<OverflowBreadcrumbProps> = ({
+    breadcrumbs,
+    selected,
+    onSelect
+}) => {
+
+    const overflowItems = React.useMemo(() => {
+        return breadcrumbs.map((breadcrumb, idx) => <>
+            <OverflowItem id={breadcrumb.id} groupId={breadcrumb.id} key={`button-items-${breadcrumb.id}`} priority={breadcrumb.id === selected ? 1 : 1}>
```

Comment:
maybe no need for the ternary on priority, if always 1

Reply:
Yes - I meant to remove that.